src/display/api.js

```diff
@@ -1540,7 +1540,14 @@ var WorkerTransport = (function WorkerTransportClosure() {
       });
       this.pageCache = [];
       this.pagePromises = [];
-      // We also need to wait for the worker to finish its long running tasks.
+      // We also need to wait for the worker
+      // to finish its long running tasks.
```

Comment:
Why this change?

Reply:
`linter` said that file length is too long

Reply:
This line has 78 characters, which is below 80.

Reply:
Yep saw that now. Will undo this :)